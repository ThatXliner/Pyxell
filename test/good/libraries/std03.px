
print filter(_<'c', "abcadeab".toArray()).join()
print map(chr(ord(_)+5), "1420".toArray()).join()
print reduce(ord(_)+_, 0, "XYZ".toArray())
