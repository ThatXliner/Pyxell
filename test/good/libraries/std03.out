abaab
6975
267
